src/FileSystem/FilesFinder.php

```diff
@@ -49,8 +59,7 @@ private function findInDirectories(array $directories, array $suffixes): array
             ->name($suffixesPattern)
             ->in($directories)
             ->exclude(['examples', 'Examples', 'stubs', 'Stubs', 'fixtures', 'Fixtures', 'polyfill', 'Polyfill'])
```

Comment:
`data`, `Data` can be included. PHPStan uses it: https://github.com/phpstan/phpstan/tree/master/tests/PHPStan/Analyser/data

Reply:
`data` are safe, but `Data` might be part of the code.

E.g. `DataTranferObjects`